debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlDdlParserTest.java

```diff
@@ -481,7 +483,7 @@ public void shouldParseCreateTableWithEnumAndSetColumns() {
 
     @Test
     public void shouldParseDefiner() {
-        String function = "FUNCTION fnA( a int, b int ) RETURNS tinyint(1) begin anything end;";
+        String function = "FUNCTION fnA( a int, b int ) RETURNS tinyint(1) begin -- anything end;";
```

Comment:
Why is this changing?

Reply:
"anything end" is an invalid query That cannot be parsed according to MySQL grammar, so i made it as a comment.